content/html/concepts/semantic-html/terms/rb/rb.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '<rb>'
+Description: 'Representing the base component in ruby annotations for East Asian text, its use has declined and it is now considered deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
```

Comment:
```suggestion
  - 'Tags'
  - 'Obsolete'
```
We have an "Obsolete" tag for cases like this where the item is depreciated.

Reply:
Great! Thank you :) Will remember that for next time.